purchasing/static/img/Pittsburgh_city_coat_of_arms.svg

```diff
@@ -0,0 +1,561 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
yes, it's in the footer

On Wed, Sep 2, 2015 at 12:28 PM, Ben Smithgall <notifications@github.com>
wrote:

> In purchasing/static/img/Pittsburgh_city_coat_of_arms.svg
> <https://github.com/codeforamerica/pittsburgh-purchasing-suite/pull/324#discussion_r38573614>
> :
>
> > @@ -0,0 +1,561 @@
> > +<?xml version="1.0" encoding="utf-8"?>
>
> Are we using this file anywhere? If not, maybe just include the .png
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforamerica/pittsburgh-purchasing-suite/pull/324/files#r38573614>
> .
>



-- 
Patrick Hammons
@hamhandedly